app/data_model/answer_store.py

```diff
@@ -151,7 +151,7 @@ def count(self, answer):
 
         return len(self.filter(answer.group_id, answer.block_id, answer.answer_id, answer.group_instance, answer.answer_instance))
 
-    def filter(self, group_id=None, block_id=None, answer_id=None, group_instance=None, answer_instance=None):
+    def filter(self, group_id=None, block_id=None, answer_id=None, group_instance=None, answer_instance=None, location=None):
         """
         Find all answers in the answer store for a given set of filter parameter matches.
         If no filter parameters are passed it returns a copy of the list of all answers.
```

Comment:
Shall we remove the :param and :return: documentation if we aren't going to add detail otherwise it's just boilerplate noise